libs/datamodel/core/src/transform/ast_to_dml/validate.rs

```diff
@@ -1001,30 +954,3 @@ impl<'a> Validator<'a> {
         Ok(())
     }
 }
-
-fn validate_name_collisions_with_map(schema: &dml::Datamodel, ast: &ast::SchemaAst, diagnostics: &mut Diagnostics) {
```

Comment:
So, um, where did this go? I'm maybe too blind when I'm reading this PR.

Reply:
It's in `attributes.rs` now, in the `// @map` blocks